conda/common/_logic.py

```diff
@@ -742,7 +747,7 @@ def sum_val(sol, objective_dict):
                     bestsol = newsol
                     bestval = objval(newsol, objective_dict)
                     hi = bestval
-                    log.trace("Bisection success, new range=(%d,%d)" % (lo, hi))
+                    log.log(TRACE, "Bisection success, new range=(%d,%d)" % (lo, hi))
```

Comment:
Should be , lo, hi (defer string formatting when trace is disabled)

Reply:
```suggestion
                    log.log(TRACE, "Bisection success, new range=(%d,%d)", lo, hi)
```
Like this?